crates/medea-coturn-telnet/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "medea-coturn-telnet"
+version = "0.1.0-dev"
+edition = "2018"
+description = "Coturn TURN server telnet client"
+authors = ["Instrumentisto Team <developer@instrumentisto.com>"]
+license = "MIT/Apache-2.0"
+documentation = "https://docs.rs/medea-coturn-telnet"
+homepage = "https://github.com/instrumentisto/medea/tree/master/crates/medea-coturn-telnet"
+repository = "https://github.com/instrumentisto/medea/tree/master/crates/medea-coturn-telnet"
+readme = "README.md"
+keywords = ["coturn", "telnet", "cli"]
+categories = ["web-programming"]
+
+[dependencies]
+async-trait = "0.1"
+bytes = "0.5"
+deadpool = "0.5"
+derive_more = "0.99"
+futures = "0.3"
+lazy_static = "1.4"
```

Comment:
Consider to use `once_cell` instead as a more general and macro-less solution (so has better IDE experience).